modules/ai/assets/js/editor/ai-excerpt.js

```diff
@@ -18,7 +18,7 @@ const AIExcerpt = ( { onClose, currExcerpt, updateExcerpt, postTextualContent }
 		credits,
 		usagePercentage: initialUsagePercentage,
 	} = useUserInfo( true );
-	const { data: newExcerpt, isLoading: isLoadingExcerpt, error, send } = useExcerptPrompt( {
```

Comment:
how it waits now for the excerpt prompt to be generate before moving forward to the edit form?

Reply:
For this case i added:

await send( { content: postTextualContent } ).finally( () => {
				setIsLoadingCombined( false );
			} );
			
Otherwise, there is a gap between the user loaded and the excerpt starting to load, causing a glitch where the edit window pops out for a fraction of a second.

Reply:
@romanserkelem ?

Reply:
You have an answer above, it still doesn't make sense?